requirements/edx/base.txt

```diff
@@ -147,7 +147,7 @@ laboratory==1.0.2         # via -r requirements/edx/base.in
 lazy==1.4                 # via -r requirements/edx/paver.txt, acid-xblock, lti-consumer-xblock, ora2
 libsass==0.10.0           # via -r requirements/edx/paver.txt, ora2
 loremipsum==1.0.5         # via ora2
-lti-consumer-xblock==2.7.1  # via -r requirements/edx/base.in
+lti-consumer-xblock==2.7.4  # via -r requirements/edx/base.in
```

Comment:
FYI @iamsobanjaved

Reply:
BTW: @iamsobanjaved It looks like this update merged in an earlier `make requirements` run.